### PR TITLE
Content Tree: ActionBar Buttons Modify Style

### LINKED ISSUE #2162: Content Tree: Custom Action Bar buttons are not sized correctly
See screenshot (from https://modus-web-components.trimble.com/?path=/docs/components-content-tree--multi-selection) showing active state here:

image

We should specify height and width of the button, or set consistent vertical/horizontal padding to fix this issue.

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
## Description

- Modified css of content ActionBar buttons
References 
Fixes #2162 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx [hunk 1]
```diff
@@ -191,8 +191,28 @@ const ActionBarTemplate = ({
...
+
+  modus-button svg {
+    width: 20px;
+    height: 20px;
+    padding-top: 3px;
+  }
+</style>
   <div id="tree-with-action-bar" style="display: flex; flex-direction: column; width: 400px;">
-    <div style="display: flex; justify-content: end; flex-wrap: wrap; margin-top: 1rem;">
+    <div class="toolbar" style="display: flex; justify-content: end; flex-wrap: wrap; margin-top: 1rem;">
```
Comment (saravananrmsmy) [inline stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx:215]: @ElishaSamPeterPrabhu 
As we have introduced the class name can we move the inline styles to internal ones.

File: stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx [hunk 2]
```diff
@@ -191,8 +191,28 @@ const ActionBarTemplate = ({
...
+    width: 24px;
+    height: 28px;
+    padding: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  modus-button svg {
+    width: 20px;
```
Comment (saravananrmsmy) [inline stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx:209]: I like the new look and feel of reduced button size, question over here how do we come up with the size for the width and height ?

**Review by saravananrmsmy — CHANGES_REQUESTED** [review]

File: stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx (hunk 1 above)
Comment (ElishaSamPeterPrabhu) [reply stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx]: Modified the css

File: stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx (hunk 2 above)
Comment (ElishaSamPeterPrabhu) [reply stencil-workspace/storybook/stories/components/modus-content-tree/modus-content-tree.stories.tsx:213]: The expected look was that it should be a square and icon in the middle.
P.S: There is no Figma for this bar

**Review by jewel-shajan — COMMENTED** [review]
The testing has been successfully completed, and everything is functioning as expected after the changes. Moved To PR.

**Review by saravananrmsmy — APPROVED** [review]

**Pull request merged**